schema.graphql

```diff
@@ -585,3 +585,19 @@ type DeployEscrowEvent implements DeployEvent @entity {
   # event params
   minCRatio: BigDecimal!
 }
+
+type CollateralToken @entity {
```

Comment:
@kibagateaux I can do it the original way you requested with an `Oracle` entity comprised of a `supportedTokens`, but I don't understand the _why_ of doing it this way. Maybe it's because I don't understand yet how it's being consumed by the frontend?

Reply:
type SupportedToken @entity {
  priceOracles: [Oracle!]!
}

type Oracle @entity {
  supportedTokens: [SupportedToken!] @ DerivedFrom "Oracle"
}

Reply:
ConfirmFeed event, just use `getOrCreateToken` to with `Oracle` as optional parameter. (Oracle?;)

Reply:
@kibagateaux The `SupportedToken` and `Oracle` entities have been added to the subgraph.

Subgraphs: 
1. [Mainnet](https://thegraph.com/hosted-service/subgraph/hepnerthomas/chainlink-feed-registry-test)
2. [Goerli](https://thegraph.com/hosted-service/subgraph/hepnerthomas/dd-test)